README.md

```diff
@@ -21,35 +21,43 @@ If you use this code or our dataset (including test set), please cite:
 }
 ```
 
-## Updated on 07.03.2023
+## Updated on 15.03.2023
 ### Installation
-First you need to clone this repository:
+First you need to clone this repository recursively (to clone the dss_crf submodule as well):
 ```
-git clone https://github.com/vnmsklnk/CVPR2020_GDNet
+git clone --recurse-submodules https://github.com/vnmsklnk/CVPR2020_GDNet
 cd CVPR2020_GDNet
 ```
 
-Then you can create a Python 3.7 virtual environment with new requirements.txt file:
+Then you need to build a docker image:
 ```
-virtualenv -p `which python3.7` venv
-source venv/bin/activate
-pip install -r requirements.txt
+docker build -t gdnet .
 ```
 
-Finally you need to install `pydensecrf` with active `venv`:
+
+### Usage
+Create a container from the built image
 ```
-git clone https://github.com/vnmsklnk/dss_crf.git
-cd dss_crf
-python setup.py install
+docker run -d --name gdnet_container gdnet
 ```
 
-### Usage
-First you need to download trained model at [here](https://mhaiyang.github.io/CVPR2020_GDNet/index.html).
+It is necessary to copy model and input to the container.
+You can download trained model [here](https://mhaiyang.github.io/CVPR2020_GDNet/index.html).
+To copy the pretrained model and input directory to the container, run
+```
+docker cp MODEL.PTH gdnet_container:/200.pth
```

Comment:
It is better to install all dependencies directly during the docker build. So it's worth adding layers with model loading to the Dockerfile. It is worth paying attention to [gdown](https://github.com/wkentaro/gdown). An example of downloading a model during build can be viewed [here](https://github.com/prime-slam/line-detection-association-dockers/blob/main/detectors/F-Clip/Dockerfile)